relay_test.go

```diff
@@ -490,6 +490,29 @@ func TestRelayConnection(t *testing.T) {
 	assert.Contains(t, err.Error(), errTest.Error(), "Unexpected error")
 }
 
+func TestSilentDrop(t *testing.T) {
+	getHost := func(call relay.CallFrame, conn relay.Conn) (relay.Peer, error) {
+		return relay.Peer{}, relay.RateLimitDropError{}
+	}
+
+	// Note: we cannot use WithTestServer since we override the RelayHosts.
+	opts := testutils.NewOpts().
+		SetServiceName("relay").
+		SetRelayHosts(hostsFunc(getHost)).
+		SetProcessPrefixes("nod", "nodejs-hyperbahn", "", "hyperbahn")
+	relay := testutils.NewServer(t, opts)
+	defer relay.Close()
+
```

Comment:
can we create a server, register echo, and ensure that it doesn't actually receive a call?